pkgs/tools/misc/grub/default.nix

```diff
@@ -52,301 +54,17 @@ assert !(efiSupport && xenSupport);
 
 stdenv.mkDerivation rec {
   pname = "grub";
-  inherit version;
+  version = "unstable-2023-06-23";
 
-  src = fetchurl {
-    url = "mirror://gnu/grub/grub-${version}.tar.xz";
-    sha256 = "sha256-t56kSvkbk9F80/6Ava5u1DdwZ4qaWuGSzOqAPrtlfuE=";
+  src = fetchFromSavannah {
```

Comment:
```suggestion
  # https://git.savannah.gnu.org/cgit/grub.git/
  src = fetchFromSavannah {
```

Reply:
We don't put similar comment on other fetchFromSavannah or fetchFromGithub + it should be easier queryable with .src.url ?

Reply:
Don't really see the point tbh.